pom.xml

```diff
@@ -59,8 +53,40 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
   </properties>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
```

Comment:
This used to be `sonatype-nexus-snapshots` not `ossrh`. So we either need to change the settings.xml on Jenkins to use ossrh or we change the ids in this pom. @ddcprg thoughts on which would be better/easier?